### PR TITLE
Reimplements GBP on TGMC

### PR DESCRIPTION
**Pull request by SplinterGP**
## About The Pull Request
Since tgstation made gbp work by actions and stuff it doenst work here anymore(meaning they split from us the GBP) so we stood without GBP, and now here i am adding it back.

i feel people have not really caring to fix bugs and i think GBP gives a bit of a incetive to that.

## Why It's Good For The REPO
Supposedly people should get more incentive to start squashing bugs and stuff. there's quite alot of annoying bugs in game people haven't been fixing and only doing refactors and more content adding without caring for them.

### CODE REVIEW

**Comment by SplinterGP** [comment]
this needs a new branch and other stuff [set up,](https://github.com/tgstation/tgstation/tree/gbp-balances) talk with me so we can figure it out before merging this.

**Comment by InterroLouis** [comment]
And then once it's merged MJP can get back to -443939 points

File: .github/gbp.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+"Logging" = 1
+"Mechanic" = -10
+"Performance" = 12
+"Priority: CRITICAL" = 20
+"Priority: High" = 15
+"Quality of Life" = 1
+"Refactor" = 10
+"Sound" = 3
+"Sprites" = 3
+"Unit Tests" = 6
```
Comment (SplinterGP) [inline .github/gbp.toml:23]: these probably need talking about.

File: .github/gbp.toml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+no_balance_label = "GBP: No Update"
+reset_label = "GBP: Reset"
```
Comment (SplinterGP) [inline .github/gbp.toml:2]: needs those label added

**Comment by BraveMole** [comment]
GBP is stupid and doesn't bring anything. Especially in TGMC.

**Pull request closed without merging**